tests/setuponlytests/conflicting_world_for_spigot_server/verify.sh

```diff
@@ -0,0 +1,5 @@
+mc-image-helper assert fileExists world/level.dat && \
```

Comment:
Might need to test the test, but it should be executing these will fail-fast so probably don't even need to chain these together with `&&`

Reply:
Unfortunately not; I [tested without `&&` and with some of the asserts negated](https://github.com/wlritchi/docker-minecraft-server/actions/runs/1770211926) and it passed, even when it shouldn't have.

We could also add a `set -e` at the top of each verify script rather than `&&`-chaining, or we could `source` the verify scripts from a shell with `set -e`.

Reply:
I thought the latter was the case, so I need to look into that.

Reply:
I found and fixed the missing `-e` 

https://github.com/itzg/docker-minecraft-server/commit/26809ef1f01654dfe001b5244c752a91461a270b